PublicDateAtUSN: 2015-03-19
Candidate: CVE-2015-2317
PublicDate: 2015-03-25 14:59:00 UTC
References: 
 https://www.djangoproject.com/weblog/2015/mar/18/security-releases/
 https://ubuntu.com/security/notices/USN-2539-1
 https://www.cve.org/CVERecord?id=CVE-2015-2317
Description:
 The utils.http.is_safe_url function in Django before 1.4.20, 1.5.x, 1.6.x
 before 1.6.11, 1.7.x before 1.7.7, and 1.8.x before 1.8c1 does not properly
 validate URLs, which allows remote attackers to conduct cross-site
 scripting (XSS) attacks via a control character in a URL, as demonstrated
 by a \x08javascript: URL.
Ubuntu-Description: 
Notes: 
Bugs: 
Priority: medium
Discovered-by: Daniel Chatfield
Assigned-to:
CVSS: 

Patches_python-django:
 upstream: https://github.com/django/django/commit/2a4113dbd532ce952308992633d802dc169a75f1 (1.7)
 upstream: https://github.com/django/django/commit/5510f070711540aaa8d3707776cd77494e688ef9 (1.6)
 upstream: https://github.com/django/django/commit/2342693b31f740a422abf7267c53b4e7bc487c1b (1.4)
upstream_python-django: released (1.4.20,1.6.11,1.7.7,1.8c1)
lucid_python-django: released (1.1.1-2ubuntu1.17)
precise_python-django: released (1.3.1-4ubuntu1.16)
trusty_python-django: released (1.6.1-2ubuntu0.8)
trusty/esm_python-django: released (1.6.1-2ubuntu0.8)
esm-infra-legacy/trusty_python-django: released (1.6.1-2ubuntu0.8)
utopic_python-django: released (1.6.6-1ubuntu2.2)
devel_python-django: released (1.7.6-1ubuntu2)
